telemetry/definitions/commonDefinitions.json

```
@@ -4375,6 +4417,33 @@
                 }
             ]
         },
+        {
+            "name": "codewhisperer_codeScanIssueIgnore",
+            "description": "Called when a code scan issue is ignored",
```

the description is a good place to explain non-obvious fields. e.g. could mention : 
```suggestion
            "description": "User ignored a code scan issue. variant=all means the user ignored all issues. ",
```